src/Console/Command/CommandHelp.php

```diff
@@ -113,7 +113,7 @@ public static function getHelpCopy()
 
 Instead of using command line options to customize the rule, you can save the
 project configuration in a <comment>.php_cs.dist</comment> file in the root directory
-of your project. The file must return an instance of ``PhpCsFixer\ConfigInterface``,
+of your project. The file must return an instance of `PhpCsFixer\\\\ConfigInterface <https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.2.1/src/ConfigInterface.php>`_,
```

Comment:
Can we use `PhpCsFixer\Console\Application::VERSION` so that we don't need to update the link manually?